keps/sig-cli/4292-kubectl-debug-custom-profile/README.md

```diff
@@ -518,7 +524,7 @@ in back-to-back releases.
 
 #### GA
 
-- Environment variable is removed altogether and feature is enabled and can not be disabled.
+- Feature gate (i.e. `KUBECTL_DEBUG_CUSTOM_PROFILE`) is locked to true and will be removed in 1.33.
```

Comment:
Don't we normally keep GA feature gates around for two releases?

Reply:
Thanks for review. You are right and updated.